private/model/api/passes.go

```diff
@@ -158,6 +160,38 @@ func (a *API) fixStutterNames() {
 	}
 }
 
+func (a *API) validateShapeNames() {
+	for _, s := range a.Shapes {
+		// name should start with an alphabet, that is ascii character 65 to 90, and 97 to 122;
+		// else skip that character
+		name := s.ShapeName
```

Comment:
It would be helpful to have unit test for this validation behavior.

In addition to leading underscores(`_`), what should the behavior be for other underscores in the shape's name. both within the shape's name and suffix?

Reply:
We validate only leading characters of a shape name.  

ex.
1. shape name: `Application_ShapeName` is considered a valid name and not renamed  
2. shape name: `__Application_ShapeName` will be renamed to `Application_ShapeName`. 
3. shape name : `&Application_ShapeName` will fail validation and we will exit with appropriate log message.